### ISSUE TITLE
docker-compose is dead, update to use docker compose complugin

### ISSUE THREAD

**Issue by fireba11** [greenstatic/bigbluebutton-exporter#123]
As of juli 2023 docker-compose will not get any more updates, so you might want to update docs to use the docker compose plugin instead.
Not really many changes required though.

**Comment by greenstatic** [comment]
You're right. Thanks for the heads up! You are welcome to contribute by creating a PR fixing the docs if you would like.
